### ISSUE TITLE
Export not working in 3.2 SR5

### ISSUE THREAD

**Issue by cdecluseau** [ClimberAB/ClimberCustomReport#43]
Hi !

### Description

The extension works great in 3.1 SR3. 
But after upgrading to 3.2 SR5 and when trying to export, we only have "Exporting Data". Nothing else.

In dev tools, I saw this : 
```
require.js:19:27132
Error: d.result is undefined
h.prototype.exportData/<@https://xxxxxx/resources/assets/client/client.js??1502182133715:62:5620
g@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:20:2335
h/<@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:20:2507
$eval@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:20:10132
$digest@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:20:8614
$evalAsync/<@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:20:10221
f@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:18:20512
hb/k.defer/c<@https://xxxxxx/resources/assets/external/requirejs/require.js?1502182133715:18:21973
```

We don't actually planned yet the upgrade to June/Septembre/Novembre. 

##### Expected behavior

The web browser prompt us for downloading the export file.

##### Actual behavior

![image](https://user-images.githubusercontent.com/34546595/33999072-b9df58be-e0e9-11e7-81e5-f19f65f38bed.png)

### Environment
```
[ ] Desktop
[X] Enterprise
```
### Versions

* Extension version: 1.1.12
* Browser: IE1 11.0.9600.18860 / Firefox 57.0.2
* Qlik Sense: 3.2 SR5

Thanks for any help

**Comment by dtribia** [comment]
I´m having the same problem, I can´t export report, I´m using QlikSense version 3.2 SR5, I updated the extension to last version posted in gitthub. however the problem still happening. The extension is realy very great but this export issue need to be fixed.

Thanks for help

**Comment by bjornalden** [comment]
Having the same problem. Export command gives "Exporting data" message but no exported data to be found. Running 4-node shared persistence QS 3.2.4 through external load balancer.
I agree it's a good extension apart from the export issue which makes it unuseful for us.

**Comment by edwardhallgren** [comment]
Fixed in version 1.2.3

**Comment by cdecluseau** [comment]
Hi ! GG ! So great ! Thanks for your reactivity !
